Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -264,6 +276,9 @@ public class PythonCompiler : SinglePassCompiler {
       }
 
       foreach (var ctor in dt.Ctors) {
+        if (ctor.IsGhost) {
+          continue;
+        }
```

Comment:
```suggestion
      foreach (var ctor in dt.Ctors.Where(ctor => !ctor.IsGhost)) {
```

Reply:
I made this change throughout (i.e., not just in the Python compiler).